src/main/java/javi/sonarqube/ComplexMethod.java

```diff
@@ -0,0 +1,33 @@
+package javi.sonarqube;
+
+import java.util.logging.*;
+
+public class ComplexMethod {
+
+    public String intToEnglishValue(int i) throws Exception {
+        System.out.println("Hello...");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)